### DISCUSSION TITLE
Profiles/presets for server + protocol quick access

### DISCUSSION THREAD

**Discussion by ghost** [mullvad/mullvadvpn-app#9124] [Feature requests & Ideas]
## Issue report

Operating system: Fedora 39 (or any PC OS)

App version:  2023.5

### Issue description

I'd like to request a sort of profile/preset feature, where you can save some quick profiles that you can click once to switch protocol, server, and a few settings at the same time. For example, you could click one profile and it will connect you to a New York OpenVPN server quickly, another for a Wireguard server in Switzerland, and a third profile/preset for a Wireguard double hop with entry in Netherlands and exit in Switzerland.

In general, just a new button or dropdown on the primary/default application screen or settings menu that we can save profiles/presets in to quickly switch between multiple frequently used sets of server + protocol combinations.

**Comment by raksooo** [comment]
Thanks for the feature request! Part of your request will be solved by our upcoming Custom lists feature, which will be part of the next app release. It allows you to create location lists with a custom set of countries/cities/servers. This wont let you easily switch other settings as well though. Profiles would indeed be a nice addition to the app.

**Comment by PaperTobi** [comment]
Hey,

Just checking in—are there any updates on the profile/preset feature? A way to save and quickly switch between different server, protocol, and settings would be super useful. Is this still being considered?

Thanks
